eclass/wine.eclass

```diff
@@ -69,6 +69,7 @@ BDEPEND="
 	)
 "
 IDEPEND=">=app-eselect/eselect-wine-2"
+RDEPEND="arm64? ( wow64? ( app-emulation/fex-xtajit ) )"
```

Comment:
One last minor nitpick, I (personally) prefer to sort RDEPEND first per skel.ebuild order rather than alphabetical when it's not inconvenient to., so I'd put it above BDEPEND.